hello-world/src/main/resources/archetype-resources/pom.xml

```diff
@@ -68,13 +68,25 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.20</version>
+            <version>2.30</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-support</artifactId>
-            <version>2.14</version>
+            <version>3.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>script-security</artifactId>
```

Comment:
This kind of thing should be in `dependencyManagement` if necessary, not as an explicit dependency.

Reply:
I've moved it to `dependencyManagement` in hopes that I've understood your intention.  I'm afraid that I don't really understand maven well enough to be sure I did it correctly.  Let me know if I've gone the wrong direction with the change.